pixart.js

```diff
@@ -1 +1,34 @@
+var brush = document.querySelector(".brush");
+var button = document.querySelector("#set-color");
+var form = document.querySelector("#form");
+var body = document.querySelector("body");
 
+function changeBrushButton(event) {
+  event.preventDefault();
+  var choice = $("#color-field").val();
+  brush.style.backgroundColor = choice;
+}
+
+function changeBrushKey(event) {
+  if (13 == event.keyCode) {
+    var choice = $("#color-field").val();
+    brush.style.backgroundColor = choice;
+  }
+}
+
+function createSquares(num) {
+  for (i = 0; i <= num; i++) {
+    div = document.createElement("div");
+    document.body.appendChild(div);
+    div.className = "square";
+    div.addEventListener("click", changeColor);
+  }
```

Comment:
:+1:  think you can take the next step and make it a real paint brush, instead of clicks?